client/src/Components/SelectionList.jsx

```diff
@@ -7,85 +7,72 @@ class SelectionList extends React.Component {
   constructor(props){
     super(props);
     this.state = {
-      selectionPossibilties : []
+      selectionPossibilties : [],
+      optionPossibilities : []
     }
-    this.getItemOptions = this.getItemOptions.bind(this);
+    this.getSelectValue = this.getSelectValue.bind(this);
+    this.getOptionsValue = this.getOptionsValue.bind(this);
   }
 
   componentDidMount() {
-      console.log(this.getItemOptions(), 'this is what getItemOptions returns')
-    this.getItemOptions();
+    this.getSelectValue();
+    this.getOptionsValue();
   }
 
-  // getItemOptions(){
-  //   var selection = [];
-  //   getItemSelection(this.props.randomItemNumber, (err, data) => {
-  //     // if (err){
-  //     //   console.log(err, "err from selection list");
-  //     // return;
-  //     // } 
-  //     if (data[0]['options']){
-  //       var itemHasSelection = data[0]['options']
-  //       for (var keys in itemHasSelection){
-  //         selection.push(keys)
-  //       }
-  //       console.log(selection[0], 'before return statement')
-  //     }
-  //     return;
-    
-  //   })
-  //   console.log(selection[0], 'after return statement')  
-  //   return selection;
-  // }
 
-  getItemOptions(){
-    console.log(this.state.selectionPossibilties)
+  getSelectValue(){
```

Comment:
I am going to place an 'overall review' comment regarding the functions `getSelectValue` and `getOptionValue`